### PR TITLE
Updates to Kontena docs

### PR DESCRIPTION
**Pull request by miskun**
Massive overhaul of Kontena docs.

### CODE REVIEW

**Comment by miskun** [comment]
@SpComb You have done some updates to docs during the past few days. I just noticed this PR is trying to overwrite your latest additions. We'll need to sort this out before we merge this.

File: docs/getting-started/installing/README.md [hunk 1]
```diff
@@ -14,29 +14,8 @@ Please see below instructions for automated installation of Kontena.
 * [UpCloud](upcloud.md)
 * [Vagrant](vagrant.md)
 
-Please see below instructions for custom Kontena installations.
+Below are instructions for custom Kontena installations.
 
 * [CoreOS](coreos.md)
 * [Docker Compose](docker-compose.md)
 * [Ubuntu 14.04](ubuntu.md)
-
```
Comment (miskun) [inline docs/getting-started/installing/README.md]: @SpComb This stuff below is needed, I believe?

File: docs/getting-started/installing/docker-compose.md [hunk 2]
```diff
@@ -4,40 +4,37 @@ title: Docker Compose
...
 
 ## Installing Kontena Master
 
-Kontena Master is an orchestrator component that manages Kontena Grids/Nodes. Installing Kontena Master using Docker Compose can be done with the following steps:
+Kontena Master is an orchestrator component that manages Kontena Grids/Nodes. Installing Kontena Master using Docker Compose can be accomplished via the following steps:
 
-**Step 1:** create `docker-compose.yml` file with the following contents:
+**Step 1:** create a `docker-compose.yml` file with the following contents:
 
-```yml
```
Comment (miskun) [inline docs/getting-started/installing/docker-compose.md]: @SpComb @jakolehm I believe something is changed in here, as well...

File: docs/core-concepts/networking.md [hunk 3]
```diff
@@ -110,41 +109,39 @@ Each host Node runs a number of infrastructure services as Docker containers, us
...
 
 The overlay network is powered by Weave Net, using [Weave's encrypted `sleeve` tunnels](https://www.weave.works/docs/net/latest/using-weave/security-untrusted-networks/) to form a flat Layer 2 network spanning all Grid Nodes and connected Containers.
 
 Alternatively, [Weave's Fast Datapath](https://www.weave.works/docs/net/latest/using-weave/fastdp/) can be used for traffic between Nodes within the Kontena Grid's [Trusted Subnets](https://kontena.io/docs/using-kontena/grids#grid-trusted-subnets).
-Using Trusted Subnets and Weave's Fast Datapath provides [improved performance](https://www.weave.works/weave-docker-networking-performance-fast-data-path/) at the cost of a lack of data plane encryption between Nodes.
+Using Trusted Subnets and Weave's Fast Datapath provides [improved performance](https://www.weave.works/weave-docker-networking-performance-fast-data-path/). The cost of this method is a lack of data plane encryption between Nodes.
 
 ## Service Containers
 
-Each Service Container has two
```
Comment (miskun) [inline docs/core-concepts/networking.md]: @SpComb can you explain this? it's broken sentence and i'm not 100% sure if the corrected version is good

**Comment by matti** [comment]
1 commit to rule them all ✊🏻

File: docs/getting-started/installing/docker-compose.md [hunk 4]
```diff
@@ -115,52 +110,17 @@ agent:
...
 
 - `KONTENA_URI` is the uri to Kontena Master (use ws:// for a non-tls connection)
-- `KONTENA_TOKEN` is the grid token, which can be acquired from the master using the `kontena grid show --token my-grid` command
+- `KONTENA_TOKEN` is the Kontena Grid token, which can be acquired from Kontena Master using the `kontena grid show my-grid` command
 - `KONTENA_PEER_INTERFACE` is the network interface that is used to connect the other nodes in the grid.
 
 **Step 2:** Run the command `docker-compose up -d`
 
-To allow Kontena agent to pull from Kontena's built-in private image registry you must add `--insecure-registry="10.81.0.0/16"` to Docker daemon options on the host machine. The most platform-independent way to do this is with the `/etc/docker/daemon.json` config file:
+To allow the Kontena Agent to pull from Kontena's built-in private image registry, you must add `--insecure-registry="10.81.0.0/19"` to the Docker daemon options on the host machine.
```
Comment (SpComb) [inline docs/getting-started/installing/docker-compose.md]: This is reverting the `/19` -> `/16` fix, and host node DNS setup instructions.

File: docs/getting-started/installing/README.md (hunk 1 above)
Comment (SpComb) [inline docs/getting-started/installing/README.md]: This is from #1056

File: docs/getting-started/installing/docker-compose.md (hunk 2 above)
Comment (SpComb) [inline docs/getting-started/installing/docker-compose.md]: This is from #1232

File: docs/core-concepts/networking.md (hunk 3 above)
Comment (SpComb) [inline docs/core-concepts/networking.md]: Looks like an editing-typo in the original commit, will fix separately.

https://github.com/kontena/kontena/commit/6c6f0f5b8108aec58dd3b39d24a49473270ae9ac#diff-558c44083a6753de31a97ec300284b27R131

File: docs/core-concepts/networking.md [hunk 5]
```diff
@@ -57,26 +56,26 @@ Each host Node has a total of four different network addresses:
 
   The public network address is resolved at startup using the `http://whatismyip.akamai.com` service, or it can be configured using `KONTENA_PUBLIC_IP`.
 
-  The Node's public addess can be used to connect to network services exposed on that host Node, including ports published by any Kontena Service Container that has been scheduled to run on that Node, including any instance of the Kontena Load Balancer.
+  The Node's public addess can be used to connect to network services exposed on that host Node. These include ports published by any Kontena Service Container that has been scheduled to run on that Node, including any instance of the Kontena Load Balancer.
   The Node's public address is also used for the Weave control and data plane connections between Nodes.
-  The Weave control and data plane ports are the only publically exposed services on a host Node in the default configuration.
+  The Weave control and data plane ports are the only publically exposed services on a host Node under the default Kontena configuration.
```
Comment (miskun) [inline docs/core-concepts/networking.md]: @SpComb this is Kontena Master configuration, right?

Comment (SpComb) [inline docs/core-concepts/networking.md]: This is about the default configuration of the kontena-agent on the host nodes.

CoreOS will presumably also expose at least SSH.

**Comment by SpComb** [comment]
The `docs/networking.md` looks fine, and my changes in #1332 seem to be preserved.

**Pull request merged**